megamek/src/megamek/common/LandAirMech.java

```diff
@@ -1692,6 +1692,12 @@ public int getThresh(int loc) {
         return getInternal(loc);
     }
 
+    @Override
+    public int getHighestThresh() {
+        // TODO: implement for StratOps Atmospheric Control Roll Errata
+        return 0;
```

Comment:
Best way to do this for LAMs? Maybe iterate over the internal locations array and return the highest value?